syncer/syncer.go

```diff
@@ -1454,7 +1455,7 @@ func (s *Syncer) handleRowsEvent(ev *replication.RowsEvent, ec eventContext) err
 		return err
 	}
 	if ignore {
-		binlogSkippedEventsTotal.WithLabelValues("rows", s.cfg.Name).Inc()
+		skipBinlogDurationHistogram.WithLabelValues("rows", s.cfg.Name).Observe(time.Since(ec.startTime).Seconds())
```

Comment:
what is this metirc used for?  it may cost a lot of time in this function?

Reply:
some users may need to ignore most of the data, you can take a look at our internal ONCALL-1243 issue.